packages/css/src/button/button.scss

```diff
@@ -1,4 +1,13 @@
 @import "../../node_modules/@utrecht/components/button/css";
+@import "../../utils/breakpoint";
+
+.amsterdam-button {
+  font-size: var(--amsterdam-button-font-size-narrow);
+
+  @media screen and (width > $amsterdam-breakpoint) {
```

Comment:
Could this be  a candidate for a `mixin` with some params? We use in multiple places

Reply:
Sure, what do you think of something like this:

```css
@mixin on-wide-screen {
  @media screen and (width > $amsterdam-breakpoint) {
    @content;
  }
}
```

Usage would be:

```css
  @include on-wide-screen {
    font-size: var(--amsterdam-button-font-size-wide);
  }
```